tests/foreman/ui/test_settings.py

```diff
@@ -228,34 +229,50 @@ def test_positive_update_login_page_footer_text_with_long_string():
 
     :CaseImportance: Medium
 
-    :CaseAutomation: notautomated
-
     :CaseLevel: Acceptance
     """
+    property_name = 'login_text'
+    property_value = [
+        ele.value
+        for ele in entities.Setting().search(query={'search': 'name={}'.format(property_name)})
+    ][0]
+    login_text_data = gen_string('alpha', 270)
+    with session:
+        try:
+            session.settings.update("name={}".format(property_name), "{}".format(login_text_data))
+            result = session.login.logout()
+            assert result["login_text"] == login_text_data
+        finally:
+            setting_cleanup(setting_name=property_name, setting_value=property_value)
 
 
-@stubbed()
 @tier3
 def test_negative_settings_access_to_non_admin():
     """Check non admin users can't access Administer -> Settings tab
 
     :id: 34bb9376-c5fe-431a-ac0d-ef030c0ab50e
 
     :steps:
+
         1. Login with non admin user
         2. Check "Administer" tab is not present
         3. Navigate to /settings
         4. Check message permission denied is present
 
-    :expectedresults: Administer -> Settings tab should not be available to
-        non admin users
+    :expectedresults: Administer -> Settings tab should not be available to non admin users
 
     :CaseImportance: Medium
 
-    :CaseAutomation: notautomated
-
     :CaseLevel: Acceptance
     """
+    login = gen_string('alpha')
+    password = gen_string('alpha')
+    entities.User(admin=False, login=login, password=password).create()
+    try:
+        with Session(user=login, password=password) as session:
+            assert session.settings.session.browser.title == 'Permission denied'
```

Comment:
```suggestion
            assert session.browser.title == 'Permission denied'
```

Reply:
Fixed, It should be session.settings.browser.title.

Reply:
Right, we expose `browser` in `session`, but first you must initialize it, which is done automatically when accessing any entity. :+1: